ci/verify-docs.sh

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
-set -e
-
-git diff --exit-code ./src/README.md
-EXIT_CODE=$?
-
-if [ "$EXIT_CODE" != "0" ]; then
-  echo "ERROR: JS API documentation not up-to-date with the source code. Follow these steps:"
-  echo " - verify if 'documentation.yml' configuration is up-to-date with the 'src' folder content,"
-  echo " - run 'npm run docs',"
-  echo " - check output 'src/README.md' file,"
-  echo " - commit updated 'src/README.md' file."
-  exit 1
-fi;
+
+checkReadmeFile() {
+    NPM_SCRIPT_SUFFIX=$1
+    README_DIRECTORY=$2
+    DOCS_CONFIG_FILE=$3
```

Comment:
Looks like magic :smile: I believe there are function calls in bash. Maybe using regular function calls would be less magical and easier to understand?

Reply:
I'm also not expert in bash, I did it this way just to name expected function arguments 🙂

Reply:
Ok, I just found out that bash does not have named arguments :smile: 
https://linuxize.com/post/bash-functions/

Sounds reasonable, then!

Reply:
Yes, unfortunately.